Program.fs

```diff
@@ -669,4 +588,82 @@ let rec main argv =
                     else
                         FemtoResult.ValidationFailed
 
-        int result
+
+type CLIArguments =
+    | [<MainCommand; ExactlyOnce>] Project of path : string
+    | Validate
+    | Resolve
+    | Preview
+
+    interface IArgParserTemplate with
+
+        member this.Usage =
+            match this with
+            | Project _ -> "specify the path to the F# project"
+            | Validate -> "check that the XML tags used in the F# project file is parsable and a npm package version can be calculated"
+            | Resolve -> "resolve package resolution and apply the required actions"
```

Comment:
Hmm it is not very important but the wording is a bit off, maybe either one of these:
 - 1) `resolve and install required packages` 
 - 2) `apply package resolution and installation` 

I like (1) more

Reply:
Good idea :)